### DISCUSSION TITLE
建议支持抖音小程序的纯原生方案

### DISCUSSION THREAD

**Discussion by MrCode612** [sonofmagic/monorepo-template#138] [General]
weapp-tailwindcss 提供了非常棒的小程序使用 tailwindcss 的解决方案，就抖音小程序而言，目前可以使用 gulp 的方式进行适配，但是纯原生的方式应该是更完美的解决方案，建议增加对抖音小程序的纯原生方式的支持。

**Comment by sonofmagic** [comment]
已经迁移到 weapp-vite 仓库:
https://github.com/weapp-vite/weapp-vite/discussions/91
